lib/src/revset.rs

```diff
@@ -2139,6 +2148,15 @@ fn resolve_commit_ref(
                 })
             }
         }
+        RevsetCommitRef::WorkingCopies() => {
+            let wc_commits: Vec<CommitId> = repo
+                .view()
+                .wc_commit_ids()
+                .iter()
+                .map(|(_, v)| v.clone())
+                .collect();
```

Comment:
nit: `.values().cloned()`?

Reply:
Also `collect_vec()` and remove the type hint. Thanks!